src/varname.jl

```diff
@@ -98,6 +101,15 @@ function Base.show(io::IO, vn::VarName)
     end
 end
 
+function Base.show(io::IO, vn::VarName{<:Any, <:Lens})
+    print(io, getsym(vn))
+    Setfield.print_application(io, vn.indexing)
+end
+
+# TODO: Should this really go here?
```

Comment:
The answer is no of course, but I'm not bothered to start impl this in a nicer way until we've decided that this is a path we want to take.